executable/optionrom.ksy

```diff
@@ -0,0 +1,474 @@
+meta:
+  id: optrom
```

Comment:
>it describes a part of firmware

Firmware is executable code, right? And optionroms are actually legacy bios modules, they are discovered and loaded by loader code in runtime, so they are more like executable file formats.

More, I guess we don't need a separate firmware folder at all. Everything is a computer, why do we discriminate them against x86?